EXILED/Exiled.API/Extensions/StringExtensions.cs

```diff
@@ -119,12 +119,12 @@ public static string ToString<T>(this IEnumerable<T> enumerable, bool showIndex
         /// <returns>Name without brackets.</returns>
         public static string RemoveBracketsOnEndOfName(this string name)
         {
-            int bracketStart = name.IndexOf('(') - 1;
```

Comment:
You could just remove the -1 part and it's would fix the issue no ? 
Also adding a remove whitespace and everything should go fine

Reply:
I'd rather not have a scenario like that occur again so replace is more of a guarantee.

Reply:
Also I am not home, I am using parsec to do these things so I am more or less doing the first thing that works

Reply:
NW don't always use Clone as name so that why so now we only remove anything after ( and trim white space :3